.gitignore

```diff
@@ -21,3 +20,6 @@ examples/*/yarn.lock
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# editors
+.idea
```

Comment:
Well, in that case lets also add
`.vscode`
`.history`

Reply:
Done!